runtime/lua/vim/ui.lua

```diff
@@ -125,12 +127,16 @@ end
 --- ```
 ---
 ---@param path string Path or URL to open
+---@param opts? table Additional options
+---     - cmd (string[]|string)
+---           Alternative URL handler: if set, editor will use this command
+---           to handle 'path' as its only command argument.
```

Comment:
single quotes in vim help files link to vim options
```suggestion
---           to handle `path` as its last command argument.
```